### PR TITLE
Update stop endpoints and response

### LINKED ISSUE #281: [API parity] Document inline=drop,pickup, dropId and pickupId on stop endpoints
Landed in the API via https://github.com/azurestandard/beehive/pull/4916.

Need to document:

- `drop` (drop object, returned only if `inline=drop` is passed and the stop is a drop)
- `pickup` (pickup object, returned only if `inline=pickup` is passed and the stop is a pickup)
- `dropId` (`int`) (always returned if stop is a drop)
- `pickupId` (`int`) (always returned if stop is a pickup)

### PR DESCRIPTION
**Pull request by igregson**
Resolves #281

### CODE REVIEW

File: spec.json [hunk 1]
```diff
@@ -9015,6 +9021,23 @@
...
         }
+      },
+      "stopInline": {
+        "name": "inline",
+        "in": "query",
+        "description": "Optional properties returned if specified in the request.",
+        "required": false,
+        "style": "form",
+        "schema": {
+          "type": "array",
```
Comment (davidmcatee-azure) [inline spec.json:9032]: `inline` is not really an array of strings, but a string that is a comma-separated list.

**Review by davidmcatee-azure — CHANGES_REQUESTED** [review]

File: spec.json (hunk 1 above)
Comment (igregson) [reply spec.json]: Argh... just kidding. It seems Redoc doesn't like the change I made. Looking into this further...

Comment (igregson) [reply spec.json]: Alright, if we specify this as a string and use `enum` for the possible values, here's how Redoc shows it: 

image

I'm not 100% sure this is what we want, as it's not clear that the string is comma-separated. If we use `array` as the type, isn't this more clear? Here's how Redoc show it is using `array`: 

image

Comment (davidmcatee-azure) [reply spec.json]: I'd be OK keeping it as an array of strings, as long as the comma-separated aspect is made clear somewhere.

Comment (igregson) [reply spec.json:9032]: As for how we want to make the comma-separated aspect clear, it seems that Redoc doesn't yet support example requests (only example responses) ([see open issue here](https://github.com/Redocly/redoc/issues/15)). 

I'm thinking the simplest thing to do for now is to specify this in the top-level `description`. That seems best via a different PR, yes?

Comment (davidmcatee-azure) [reply spec.json:9032]: Sounds good.

**Review by davidmcatee-azure — APPROVED** [review]

**Pull request merged**